### PR TITLE
Add missing before/after Cucumber functions

### PR DESCRIPTION
**Pull request by ryantmer**
These are present elsewhere, but appear to have been missed in these two files.

## Proposed changes

Filling in some missing fields in example and helper files.

## Types of changes

What types of changes does your code introduce to WebdriverIO?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

## Further comments

(None)

### CODE REVIEW

**Comment by ryantmer** [comment]
Looks like Sauce had some trouble connecting - I'd imagine because this is coming from a fork?

File: lib/helpers/wdio.conf.ejs [hunk 1]
```diff
@@ -295,6 +295,42 @@ exports.config = {
     // afterSuite: function (suite) {
     // },
     /**
+     * Runs before a Cucumber feature
```
Comment (christian-bromann) [inline lib/helpers/wdio.conf.ejs]: Can we check if the framework is Cucumber and only include these when it is (and also not include `beforeTest` and other Mocha/Jasmine like hooks

Comment (ryantmer) [inline lib/helpers/wdio.conf.ejs]: Just to clarify, from poking around it seems that the Mocha/Jasmine ones are before/after Suite, Hook, and Test, is that correct?

Comment (christian-bromann) [inline lib/helpers/wdio.conf.ejs]: Yes

**Comment by ryantmer** [comment]
Updated, tried to match the ordering that was already there (which seems to be ~test-chronological).

**Pull request merged**